README.md

```diff
@@ -1,91 +1,39 @@
-# CleverHans (latest release: v3.1.0)
-
-Note: This is the final version of CleverHans v3, which supports TensorFlow 1, Python 3 and Python 2. In our version 4 update, we’re dropping support for TF1 and Python 2, and transitioning to JAX, PyTorch and TF2 on Python 3.
-
-This version was last tested with Python 3.5 and Tensorflow {1.8, 1.12} on Ubuntu 14.04.5 LTS (Trusty Tahr). You’re welcome to continue using it, but it’s no longer maintained.
+# CleverHans (latest release: v4.0.0)
 
 <img src="https://github.com/tensorflow/cleverhans/blob/master/assets/logo.png?raw=true" alt="cleverhans logo">
 
-[![Build Status](https://travis-ci.org/tensorflow/cleverhans.svg?branch=master)](https://travis-ci.org/tensorflow/cleverhans)
```

Comment:
Will the new CI also show its status?

Reply:
Yes, I've removed it for now until we get the new CI up and running, but I want to add it back when it's ready.